xonsh/lexer.py

```diff
@@ -43,12 +43,13 @@
 for (op, type) in _op_map.items():
     token_map[(tokenize.OP, op)] = type
 
-token_map[tokenize.NUMBER] = 'NUMBER'
 token_map[tokenize.STRING] = 'STRING'
 token_map[tokenize.NEWLINE] = 'NEWLINE'
 token_map[tokenize.INDENT] = 'INDENT'
 token_map[tokenize.DEDENT] = 'DEDENT'
 
+_REDIRECT_NAMES = {'out', 'err', 'all', 'o', 'e', 'a'}
```

Comment:
Yep.  I can change that.  There are several other things in built_ins.py that could also be frozensets.